packages/sdk/src/integration/server/index.ts

```diff
@@ -78,16 +78,21 @@ export const integrationHandler =
           throw new Error(`Unknown operation ${ctx.operation}`)
       }
       return response ? { ...response, status: response.status ?? 200 } : { status: 200 }
-    } catch (thrown) {
-      if (isApiError(thrown)) {
-        const runtimeError = new RuntimeError(thrown.message, thrown)
+    } catch (error) {
+      if (isApiError(error)) {
+        if (error.type === 'UpstreamProvider') {
```

Comment:
**question:** Have you spoken with the stratus team? Do you know why we need to map API errors to Runtime errors ?

Reply:
not yet, as discussed I'll first bring this up tomorrow in our know share meeting